packages/@rnw-scripts/just-task/just-task.js

```diff
@@ -65,5 +65,5 @@ if (hasE2eTests && hasUnitTests) {
 } else if (hasUnitTests) {
   task('test', 'unitTest');
 } else {
-  logger.info('No tests found');
+  task('test', () => logger.info('No tests found'));
```

Comment:
Heh, looks like we were previously printing out "no tests found" for packages without tests when we were doing any task (e.g. building)